source/tests/system/rfmx_macros.h

```diff
@@ -48,11 +44,11 @@
 #define EXPECT_ERROR(expected_error_, message_substring_, session_, response_call_) \
```

Comment:
I'd prefer to not make that change because for one, there would be several variants such as validating exact string versus substring. But the main reason I don't want this at this time is that I'd rather make a method that *returns* the caught exception (such as `TestAssert.Throws` in ASW) so we can more clearly separate the 'act' and 'assert' portions of the test.